downstream/modules/platform/proc-setup-postgresql-ext-database.adoc

```diff
@@ -47,111 +49,92 @@ If so, connection string parameters override any conflicting command line option
+pg_sslmode=verify-full or verify-ca
+
+# {GatewayStart}
+automationgateway_pgclient_sslcert=/path/to/gateway.cert
+automationgateway_pgclient_sslkey=/path/to/gateway.key
+automationgateway_pg_sslmode=verify-full or verify-ca
+
+# {HubNameStart}
+automationhub_pgclient_sslcert=/path/to/pulp.cert
+automationhub_pgclient_sslkey=/path/to/pulp.key
+automationhub_pg_sslmode=verify-full or verify-ca
+	
+# {EDAName}
+automationedacontroller_pgclient_sslcert=/path/to/eda.cert
+automationedacontroller_pgclient_sslkey=/path/to/eda.key
+automationedacontroller_pg_sslmode=verify-full or verify-ca
+----
 
-# {HubNameStart} database variables
+== Optional: Using custom TLS certificates
 
-# automationhub_pg_host=data.example.com 
-# automationhub_pg_port=<port_number> 
-# automationhub_pg_database=<database_name> 
-# automationhub_pg_username=<username>
-# automationhub_pg_sslmode=verify-full # This can be either verify-ca or verify-full
-# automationhub_pgclient_sslcert=/path/to/cert # Path to the certificate file 
-# automationhub_pgclient_sslkey=/path/to/key # Path to the key file
+By default, the installation program generates self-signed TLS certificates and keys for all {PlatformNameShort} services.
 
+If you want to replace these with your own custom certificate and key, then set the following inventory file variables:
 
-# {GatewayStart} database variables
+[source,yaml,subs="+attributes"]
+----
+aap_ca_cert_file=<path_to_ca_tls_certificate>
+aap_ca_key_file=<path_to_ca_tls_key>
+----
 
-# automationgateway_install_pg_host=data.example.com 
-# automationgateway_install_pg_port=<port_number> 
-# automationgateway_install_pg_database=<database_name> 
-# automationgateway_install_pg_username=<username>
-# automationgateway_pg_sslmode=verify-full # This can be either verify-ca or verify-full
-# automationgateway_pgclient_sslcert=/path/to/cert # Path to the certificate file 
-# automationgateway_pgclient_sslkey=/path/to/key # Path to the key file
+If any of your certificates are signed by a custom Certificate Authority (CA), then you must specify the Certificate Authority’s certificate by using the `custom_ca_cert` inventory file variable:
+[source,yaml,subs="+attributes"]
+----
+custom_ca_cert=<path_to_custom_ca_certificate>
 ----
-+
-. Run the installation program. If you are using a PostgreSQL database, the database is owned by the connecting user and must have a `createDB` or administrator role assigned to it.
-. Check that you can connect to the created database with the credentials provided in the inventory file.
-. Check the permission of the user. The user should have the `createDB` or administrator role.
 
 [NOTE]
 ====
-During this procedure, you must check the External Database coverage. For further information, see https://access.redhat.com/articles/4010491
+Note: If you have more than one custom CA certificate, combine them into a single file, then reference the combined certificate with the `custom_ca_cert` inventory file variable.
```

Comment:
You can remove the Note: here because the tagging will add the text automatically.

Reply:
Good catch, @dcdacosta. Incorporated your feedback.